src/api/accounts/accounts.md

```diff
@@ -32,17 +32,23 @@ individual account.
 ### Account
 
 
-|   Field    |        Type        |                         Description                         |
-| :--------- | :----------------- | :---------------------------------------------------------- |
-| id         | String             | The unique identifier.                                      |
-| type       | String             | Account type, must be either 'org' or 'individual'.         |
-| name       | String             | The display name of the Account.                            |
-| region     | String             | The region that the Account will operate in.                |
-| test       | Boolean            | A flag which is only present if the Account is for testing. |
-| createdAt  | {% dt Timestamp %} | When the Account was created.                               |
-| modifiedAt | {% dt Timestamp %} | When the Account was updated.                               |
-| createdBy  | {% dt CRN %}       | The User or API Key that created the Account.               |
-| modifiedBy | {% dt CRN %}       | The User or API Key that updated the Account.               |
+|     Field     |        Type        |                         Description                         |
+| :------------ | :----------------- | :---------------------------------------------------------- |
+| id            | String             | The unique identifier.                                      |
+| type          | String             | Account type, must be either 'org' or 'individual'.         |
+| name          | String             | The display name of the Account.                            |
+| region        | String             | The region that the Account will operate in.                |
+| test          | Boolean            | A flag which is only present if the account is for testing. |
```

Comment:
I think Account here should have a capital A like it previously did, as it is a resource.

Reply:
Yep, got removed when resolving a merge conflict.